pcdsdevices/epics_motor.py

```diff
@@ -51,6 +51,8 @@ class PCDSMotorBase(FltMvInterface, EpicsMotor):
     # Disable missing field that our EPICS motor record lacks
     # This attribute is tracked by the _pos_changed callback
     direction_of_travel = Component(Signal)
+    # This attribute will show the current status of motor
+    current_motor_status = Component(EpicsSignal, ".DISP")
```

Comment:
`current_motor_status` is a little misleading. I think the field is just here for disabling the motor. I would be tempted to call it what it is -> `disabled`.